README.md

```diff
@@ -1,13 +1,16 @@
-# Docker Client [![Circle CI](https://circleci.com/gh/spotify/docker-client.png?style=badge)](https://circleci.com/gh/spotify/docker-client) [![codecov.io](https://codecov.io/github/spotify/docker-client/coverage.svg?branch=master)](https://codecov.io/github/spotify/docker-client?branch=master)
-
+# Docker Client [![Build Status](https://travis-ci.org/spotify/docker-client.svg?branch=master)](https://travis-ci.org/spotify/docker-client) [![codecov.io](https://codecov.io/github/spotify/docker-client/coverage.svg?branch=master)](https://codecov.io/github/spotify/docker-client?branch=master)
 
 This is a simple [Docker](https://github.com/docker/docker) client written in Java.
+We build and test docker-client on Docker API versions 1.6 - 1.10 (specifically the ones (here)[1]).
```

Comment:
nitpick: the API versions have different values than the actual client/server version for some reason: https://docs.docker.com/engine/reference/api/docker_remote_api/

so we should refer to either "Docker version" or "Docker API version" and use the appropriate column in the table on that page